MainTest.java

```diff
@@ -0,0 +1,20 @@
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class MainTest {
+  
+      @Test
+      public void testNothing(){}
+
+      @Test
+      public void testSomeStuff(){
+        /*if (assertFalse(assertEquals((2+3),5))) System.out.println("Dies ist eine falsche Aussage");*/
+
+        assertTrue(2+3==5);
```

Comment:
hier kann man natürlich auch assertEquals verwenden.
Sie können und sollten als ersten Parameter eine "aussagekräftige Meldung" angeben, die ausgegeben wird, wenn der Test fehlschlägt: 
Das finden Sie z.B. in der Dokumentation zu JUnit asserts:

assertEquals(String message, long expected, long actual)